.env.development

```diff
@@ -0,0 +1,4 @@
+FACEBOOK_API_KEY: 830527976957438
```

Comment:
I expect that API keys and secrets should not be included as part of this pull request.

Reply:
Development is stopped. I already noticed above all parts are not ported. Please finish it by yourself or by another enthusiast.